Sitemap.php

```diff
@@ -166,7 +171,7 @@ private function createNewFile()
         // Use XML stylesheet, if available
         if (isset($this->stylesheet)) {
             $this->writer->writePi('xml-stylesheet', "type=\"text/xsl\" href=\"" . $this->stylesheet . "\"");
-            $this->writer->writeRaw("\n");            
+            $this->writer->writeRaw(PHP_EOL);
```

Comment:
That would result in CRLF in generated sitemaps under Windows. That was originally not not desirable. Is there a reason for the change?